src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/MessageService.java

```diff
@@ -84,13 +89,21 @@ public String sendOrRespondMessage(final String profile, final Integer projectId
         .password(rpaAPIConfig.getBuyerPwd()).ittCode(procurementEvent.getExternalReferenceId())
         .broadcastMessage(nonOCDS.getIsBroadcast() ? "Yes" : "No").messagingAction(CREATE_MESSAGE)
         .messageSubject(ocds.getTitle()).messageBody(ocds.getDescription())
-        .messageClassification(nonOCDS.getClassification().getValue())
-        .senderName(buyerUser.get().getName()).supplierName("").messageReceivedDate("");
+        .messageClassification(nonOCDS.getClassification().getValue()).senderName("")
+        .supplierName("").messageReceivedDate("");
 
     // To reply the message
     if (nonOCDS.getParentId() != null) {
-      // TODO get message details using parentId
-      inputBuilder.messagingAction(RESPOND_MESSAGE).messageReceivedDate("message-receive-date");
+      DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern(DATE_FORMATTER);
```

Comment:
Can we make the formatter instance a class constant - e.g.
`static final DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'.000+00:00'")`
Do any of the predefined formatters meet the required format spec?

Reply:
Created formatter instance at class level.